notification-sender/notification_sender.bal

```diff
@@ -83,45 +82,53 @@ public function main(string... args) {
 }
 
 # Send notification to the customers.
+#
 # + return - State of whether the process of sending notification is success or not
```

Comment:
Shall we rephrase this to say `A boolean indicating the status of the send notification process`

Reply:
Fixed with https://github.com/ballerina-guides/gmail-spreadsheet-integration/pull/20/commits/231df90d595dfb0824c5eaee49acb73cfb6451e6